probe/scripts/run_docker.sh

```diff
@@ -3,13 +3,12 @@ workspace_root=$(realpath "${script_dir}/../")
 
 # Docker image information.
 #docker_image_with_tag="kindlingproject/kindling-compiler"
-docker_image_with_tag="kindlingproject/kindling-compiler"
+# docker_image_with_tag="kindlingproject/kindling-compiler"
+docker_image_with_tag="kindlingproject/kindling-compiler:dev"
```

Comment:
I think this is unnecessary because once your PR is merged, we will update the image using the tag `latest`.